src/Utils/StringHelper.php

```diff
@@ -30,20 +30,20 @@ public static function ensureMaxLength(?string $string, int $length): ?string
     public static function sanitizeDDE(string $text): string
     {
         // see #3189
-        if (\strlen($text) === 0) {
+        if ($text === '') {
```

Comment:
as the method receives a string, this should never be null. what does it change?

Reply:
This is a micro optimization because comparing pure strings is faster than calling a method